pkg/config/image_watcher.go

```diff
@@ -25,12 +32,78 @@ type ImageWatcherTarget struct {
 	Field    string `json:"field"`
 }
 
-// LoadImageWatcher finds the config files for the image watcher in the .pipe directory first up.
-// And returns parsed config, False is returned as the second returned value if not found.
+// LoadImageWatcher finds the config files for the image watcher in the .pipe
+// directory first up. And returns parsed config after merging the targets.
+// Only one of includes or excludes can be used.
+// False is returned as the second returned value if not found.
 func LoadImageWatcher(repoRoot string, includes, excludes []string) (*ImageWatcherSpec, bool, error) {
-	// TODO: Load image watcher config
-	//   referring to AnalysisTemplateSpec
-	return nil, false, nil
+	dir := filepath.Join(repoRoot, SharedConfigurationDirName)
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return nil, false, fmt.Errorf("failed to read %s: %w", dir, err)
```

Comment:
I think no `.pipe` directory is a normal case. In that case, is piped sending this error to the log system?

Reply:
Exactly, it should be handled as not found case.

Reply:
Maybe we should compare with `os.IsNotExist`.

Reply:
Right